### DISCUSSION TITLE
Training: Missing Flags

### DISCUSSION THREAD

**Discussion by SalvaDC-DecoIT** [mitre/caldera#2711] [Q&A]
In order to capture the flag for training\advanced\Add new user, I terminated the Caldera and re-run it again.  After new run (I did not use the --fresh flag) I missed all captured flags. 
Has anyone encountered this before?

**Comment by UsrBern** [comment]
Necroing this because I'm having the same issue. Did you ever figure out how to get that flag?

**Comment by HackedRico** [comment]
Thanks for resurfacing this. There are two separate issues here:

1. Starting CALDERA with `--fresh` resets Training progress. This is documented in the proposed README update in [training#195](https://github.com/mitre/training/pull/195).

2. The Advanced **Add new user** flag had a separate verification bug. CALDERA now stores configured passwords as Argon2 hashes, but the flag compared the stored value directly with the plaintext password `test`. That comparison can never succeed once passwords are hashed.

The proposed fix is in [training#197](https://github.com/mitre/training/pull/197). It uses CALDERA’s existing `verify_hash` helper and adds regression tests. No packages need to be installed.

Since you noted that `--fresh` was not used, the README guidance alone does not explain why all previously captured flags disappeared after a normal restart. Please try the fix **Add new user** flag. If progress still disappears afterward, please share the CALDERA version, startup command, and relevant server logs so that behavior can be investigated separately.
